src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,53 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_ARRAY = 10;
+    private final K [] keys = (K[]) new Object[MAX_SIZE_ARRAY];
+    private final V [] values = (V[]) new Object[MAX_SIZE_ARRAY];
+    private int storageIndex = -1;
+
+    public StorageImpl() {
+    }
+
+    private int getFoundIndex(K[] keyArray, K key, int lengthKeyArray) {
```

Comment:
this method only works with key array, so we can remove it and its length from the parameters
```suggestion
    private int getIndexForKey(K key) {
```